index/scorch/scorch.go

```diff
@@ -100,7 +101,25 @@ func NewScorch(storeName string,
 		nextSnapshotEpoch:    1,
 		closeCh:              make(chan struct{}),
 		ineligibleForRemoval: map[string]bool{},
+		segPlugin:            defaultSegmentPlugin,
 	}
+
+	// check if the caller has requested a specific segment type/version
+	forcedSegmentVersion, ok := config["forceSegmentVersion"].(int)
+	if ok {
+		forcedSegmentType, ok2 := config["forceSegmentType"].(string)
+		if !ok2 {
+			return nil, fmt.Errorf(
+				"forceSegementVersion set to %d, must also specify forceSegmentType", forcedSegmentVersion)
```

Comment:
typo  in forceSegementVersion?  (forceSegmentVersion ?)

Reply:
Addressed in latest commit.